schemas/context/placecontext.schema.json

```diff
@@ -23,8 +23,17 @@
         "xdm:geo": {
           "title": "Geo",
           "$ref": "https://ns.adobe.com/xdm/common/geo",
-          "description":
-            "The geographic location where the experience was delivered ."
+          "description": "The geographic location where the experience was delivered."
+        },
+        "xdm:poiInteraction": {
+          "title": "POI Interaction",
+          "$ref": "https://ns.adobe.com/xdm/context/poi-interaction",
+          "description": "The Point of Interest(POI) interaction details."
```

Comment:
POI can also mean other things in other contexts.  This should not be in the xdm namespace, but in a specific "point-of-interest" one

Reply:
Can you provide an reference to an authoritative schema for POI events? I am not aware of any.